src/Command/CreateClientCommand.php

```diff
@@ -108,6 +113,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
     private function buildClientFromInput(InputInterface $input): Client
     {
+        $name = $input->getArgument('name');
```

Comment:
Maybe we can make the name optional? And if the name isn't defined, it could fall back to the identifier. WDYT?

Reply:
no strong opinion, but prefer to keep it explicit.

Reply:
Me neither, let's see if it's relevant by using it

Reply:
this is relevant in most cases, when logging with GitHub to another project, usually GitHub shows you the name/description/image of the OAuth client so you know what you are getting into, the league oauth2-server doesn't support additional metadata, but we can also solve the problem for people wanting to add things like images and description by allowing them to extend the entities ( which is not possible right now )

Reply:
ideas welcome 😛 currently using this bundle in a project, name is enough for now as i can map it to a constant array of metadata, but it's not really a good solution for the long-run;